connectors/sync_job_runner.py

```diff
@@ -143,20 +167,28 @@ async def _sync_done(self, sync_status, sync_error=None):
             "indexed_document_count": result.get("indexed_document_count", 0),
             "indexed_document_volume": result.get("indexed_document_volume", 0),
             "deleted_document_count": result.get("deleted_document_count", 0),
-            "total_document_count": await self.connector.document_count(),
         }
 
-        if sync_status == JobStatus.ERROR:
-            await self.sync_job.fail(sync_error, ingestion_stats=ingestion_stats)
-        elif sync_status == JobStatus.SUSPENDED:
-            await self.sync_job.suspend(ingestion_stats=ingestion_stats)
-        elif sync_status == JobStatus.CANCELED:
-            await self.sync_job.cancel(ingestion_stats=ingestion_stats)
-        else:
-            await self.sync_job.done(ingestion_stats=ingestion_stats)
+        if await self.reload_sync_job():
+            if await self.reload_connector():
```

Comment:
What will be "else" in this case? Do we want to log something in "else"?

Reply:
Make sense. Will add a log.

Reply:
Are you planning to add a log line here?

Reply:
I added the log to `reload_connector` method:

https://github.com/elastic/connectors-python/blob/06325cc0e563317fa296dcc05d9f2c7efa887353/connectors/sync_job_runner.py#L264-L272

But I just realized it won't always log when it returns `False`. I will add a log soon.